audit/projects/k8s-infra-prow-build-trusted/iam.json

```diff
@@ -32,7 +32,8 @@
     },
     {
       "members": [
-        "group:k8s-infra-cluster-admins@kubernetes.io"
+        "group:k8s-infra-cluster-admins@kubernetes.io",
+        "serviceAccount:prow-deployer@k8s-infra-prow-build-trusted.iam.gserviceaccount.com"
```

Comment:
This was https://github.com/kubernetes/k8s.io/pull/2148#issuecomment-856385777

Hoping to replace with a custom role (starting with addition of the role in https://github.com/kubernetes/k8s.io/pull/2156)